src/modules/dash/components/Confirmation.vue

```diff
@@ -4,31 +4,43 @@ defineProps({
   item: String,
   modal: Boolean,
 });
+
+defineEmits(["close", "delete-item", "update-item"]);
 </script>
 
 <template>
   <v-card v-if="modal" class="container d-flex w-100 h-100">
     <v-card class="w-25 ma-auto bg-white elevation-10">
-      <v-card-text>
-        Confirmar {{ action == 'delete' ? 'deleção' : 'edição'  }} do item: <b>{{ item }}</b
-        >?
+      <header class="d-flex justify-center bg-red">
+        <h3>AVISO</h3>
+      </header>
+      <v-card-text class="d-flex justify-center">
+        <p>Confirmar {{ action == 'delete' ? 'deleção' : 'edição'  }} do item: <b> {{ item }}</b
+        >?</p>
       </v-card-text>
       <v-card-actions>
-        <v-btn color="black" class="w-50" @click="this.$emit('close')"
+        <v-btn color="black" class="w-50 buttonCanc" @click="$emit('close')"
           >Cancelar</v-btn
         >
-        <v-btn color="black" class="w-50" @click="action == 'delete' ? this.$emit('delete-item') : this.$emit('update-item')"
+        <v-btn color="black" class="w-50 buttonConf ml-1" @click="action == 'delete' ? $emit('delete-item') : $emit('update-item')"
           >Confirmar</v-btn
         >
       </v-card-actions>
     </v-card> </v-card
-  >I
+  >
 </template>
 
 <style scoped>
 .container{
     position: fixed;
     top: 0; right: 0;
     background-color: rgb(0, 0, 0, 0.2);
+    opacity: 0.95;
 }
+
+.buttonConf :hover {
```

Comment:
não seria junto? se separar quer dizer que é pra ouvir hover dos elementos filhos
```suggestion
.buttonConf:hover {
```

Reply:
Corrigido